api/openapi.yaml

```diff
@@ -66,11 +66,13 @@ paths:
       operationId: deleteFile
       parameters:
       - description: A unique File identifier.
+        explode: true
```

Comment:
@shashaty where did these `explode` and `style` parameters come from? it isn't part of the spec in front and it wasn't there before the rebase